informer/collector.go

```diff
@@ -9,10 +9,21 @@ import (
 )
 
 var (
-	description *prometheus.Desc = prometheus.NewDesc(
+	creationTimestampDescription *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName("operatorkit", "informer", "creation_timestamp"),
+		"DeletionTimestamp of watched objects.",
+		[]string{
+			"kind",
+			"name",
+			"namespace",
+		},
+		nil,
+	)
+	deletionTimestampDescription *prometheus.Desc = prometheus.NewDesc(
 		prometheus.BuildFQName("operatorkit", "informer", "deletion_timestamp"),
 		"DeletionTimestamp of watched objects.",
 		[]string{
+			"kind",
```

Comment:
We also have the kind now which is something like `CertConfig` or `DrainerConfig`.